Loop/Managers/DoseMath.swift

```diff
@@ -81,28 +83,47 @@ extension InsulinCorrection {
         }
     }
 
-    /// Determins the bolus needed to perform the correction
+    /// Determines the bolus needed to perform the correction, subtracting any insulin already scheduled for
+    ///  delivery, such as the remaining portion of an ongoing temp basal.
     ///
     /// - Parameters:
     ///   - pendingInsulin: The number of units expected to be delivered, but not yet reflected in the correction
     ///   - maxBolus: The maximum allowable bolus value in units
-    ///   - volumeRounder: The smallest fraction of a unit supported in bolus delivery
+    ///   - volumeRounder: Method to round computed dose to deliverable volume
     /// - Returns: A bolus recommendation
-    fileprivate func asBolus(
+    fileprivate func asManualBolus(
         pendingInsulin: Double,
         maxBolus: Double,
         volumeRounder: ((Double) -> Double)?
-    ) -> BolusRecommendation {
+    ) -> ManualBolusRecommendation {
         var units = self.units - pendingInsulin
         units = Swift.min(maxBolus, Swift.max(0, units))
         units = volumeRounder?(units) ?? units
 
-        return BolusRecommendation(
+        return ManualBolusRecommendation(
             amount: units,
             pendingInsulin: pendingInsulin,
             notice: bolusRecommendationNotice
         )
     }
+    
+    /// Determines the bolus amount to perform a partial application correction
+    ///
+    /// - Parameters:
+    ///   - partialApplicationFactor: The fraction of needed insulin to deliver now
+    ///   - maxBolus: The maximum allowable bolus value in units
```

Comment:
[nit] align parameter name

Reply:
I'm not sure I understand. The parameters are normally indented in this form, from what I see as the pattern.